content/03.categorize.md

```diff
@@ -231,6 +231,58 @@ a need for better pathologist-computer interaction techniques that will allow
 combining the power of deep learning methods with human expertise and lead to
 better-informed decisions for patient treatment and care.
 
+### Text applications in healthcare
+
+Due to the rapid growth of scholarly publications and electronic medical/health records, biomedical text mining has become increasingly important in recent years.
+The main tasks in biological and clinical text mining include, but are not limited to, named entity recognition, relation/event extraction, and information retrieval (Figure 1).
+
+![Deep learning applications, tasks and models based on NLP perspectives.](images/biotm.png)
```

Comment:
Use dynamic figure reference. Above you will do `(Figure @fig:biotm)`

Then change the figure embed line like:

```md
![
Deep learning applications, tasks and models based on NLP perspectives.
](images/biotm.png){#fig:biotm width="100%"}
```

The figure caption will become:

> Figure 1. Deep learning applications, tasks and models based on NLP perspectives

Is this caption sufficiently detailed?